docs/quick-starts/bandit-quickstart.md

```diff
@@ -164,7 +164,7 @@ If the actions don't have attributes for the context, you can simply provide a s
 ```java
 Set<String> actions = Set.of("dog", "cat", "bird", "goldfish");
 
-Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, subjectAttributes, actions);
+Optional<String> banditAssignment = eppoClient.getStringAssignment(subjectKey, flagKey, defaultVariation, subjectAttributes, actions);
```

Comment:
@aarsilv heads up that I added defaults for the Java SDK. This will be inaccurate until there is a Java v3, but hopefully doesn't create too much confusion

Reply:
Or not; I've pulled out these changes into https://github.com/Eppo-exp/eppo-docs/pull/339. We can apply them when the Java SDK is updated to v3.